scripts/qa_crowbarsetup.sh

```diff
@@ -2047,6 +2047,8 @@ function reboot_nodes_via_ipmi
                 sleep $((50 + RANDOM % 20))
 
                 ipmitool -H $ip -U root -P $pw chassis bootdev pxe
+                # Please do not touch this sleep. It is important for the ipmi to work on qa hardware
+                sleep $((50 + RANDOM % 20))
```

Comment:
sorry for the uninformed question, but is there so status that we can query here, to do some kind of `wait_for`?